safe_web_api_playground/README.md

```diff
@@ -0,0 +1,22 @@
+#### SAFE browser website development tool
+
+This package is meant to help people that desire to create websites for the SAFE network, especially those sites that need to utilize the SAFE network web API, available on the global `window` object.
+
+Instead of having to upload your site to the network every time you want to test some code, simply use this package to experiment.
+
+
+###### SETUP
+
+- clone this repository and `cd` into the respective directory
+- Run `npm install -g gulp`
+- Run `npm install`
+- Run `bower install`
+- Run `gulp` to start server
```

Comment:
It seems we are missing a `.env` file since I get this error:
```
[13:22:30] Starting 'js'...
[13:22:30] 'js' errored after 163 μs
[13:22:30] Error: ENOENT: no such file or directory, open '.env'
```